lib/std/uuid.zig

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2015-2020 Zig Contributors
+// This file is part of [zig](https://ziglang.org/), which is MIT licensed.
+// The MIT license requires this copyright notice to be included in all copies
+// and substantial portions of the software.
+
+const rand = @import("rand.zig");
+const mem = @import("mem.zig");
+
+pub const UUID = struct {
+    id: [36]u8,
```

Comment:
a UUID should just be a `u128`.

You can add a `format` method that stringifies it.